package.json

```diff
@@ -3,6 +3,8 @@
   "version": "0.1.0",
```

Comment:
**[OPTIONAL]**

- [ ] It would be nice if you remove your live demo section since you don't have any live link for your application 👍 to make your readme descriptive and professional 😊 Once you would have a live link then you can add this live demo section again in your readme file 💯